src/services/ui/src/components/Opensearch/Filtering/FilterChips.tsx

```diff
@@ -12,17 +14,8 @@ interface RenderProp {
   clearFilter: (filter: OsFilterable, valIndex?: number) => void;
 }
 
-const checkMultiFilter = (filters: OsFilterable[]) => {
-  return (
-    filters.length > 1 ||
-    filters.some(
-      (filter) => Array.isArray(filter.value) && filter.value.length > 1
-    )
-  );
-};
-
-// render function for simple date range chips
-const renderDateChip: FC<RenderProp> = ({
+// simple date range chips
+const DateChip: FC<RenderProp> = ({
```

Comment:
In retrospect, no idea why I went with render functions over components. Looks nice.